spec/requests/api/v1/helprequest_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::HelpRequestsController, type: :request do
+  describe 'POST /v1/help_requests' do
+    it 'should collect call for help message from mobile client' do
+
+      params = {
+        title: 'Burning',
+        message: 'Pants is burning',
+        status: true
+                }
+
+      post "/api/v1/help_requests", params
```

Comment:
THB, I can't see how this is working, I would write this as 
```ruby
post "/api/v1/help_requests", { params: params }
```
Didn't you get any errors or warnings when running your tests?